sql/moz-fx-cjms-prod-f3c7/cjms_bigquery/refunds_v1/metadata.yaml

```diff
@@ -11,3 +11,15 @@ scheduling:
   # destination is the whole table, not a single partition,
   # so don't use date_partition_parameter
   date_partition_parameter: null
+  # Adjust the execution delta for the task dependencies in the bqetl_subplat_hourly DAG
+  # so this depends on the last hourly DAG run for the same date.
+  depends_on:
+    - dag_name: bqetl_subplat_hourly
+      task_id: stripe_external__charge__v1
+      execution_delta: -21h45m
+    - dag_name: bqetl_subplat_hourly
+      task_id: stripe_external__invoice__v1
+      execution_delta: -21h45m
+    - dag_name: bqetl_subplat_hourly
+      task_id: stripe_external__refund__v1
+      execution_delta: -21h45m
```

Comment:
It's annoying this is necessary. The execution deltas generated by default (1h15m) are sensible from the perspective of the run date (i.e. data interval end), but it turns out Airflow actually applies the execution deltas to the execution date (i.e. data interval start).

I think a better ultimate solution would be leveraging `ExternalTaskSensor`'s `execution_date_fn` parameter to dynamically calculate the dependency's execution date to wait for based on actual cron logic and configurable criteria (e.g. most recent run, first/last run in same date), but I didn't want to fall down that rabbit hole in this PR.

Reply:
It might be possible to use timetables to mess with the interval https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/timetable.html#crontriggertimetable

e.g. run hourly but with a 24 hour data interval.  Execution delta works with the `logical_date` property though and I'm not sure how that's affected relative to the interval.  It feels a bit hacky though and any other dependent dags would need to account for it.

Current way is fine though and I don't want to over-engineer it

Reply:
Using timetables is an interesting idea. 🤔 

> I think a better ultimate solution would be leveraging `ExternalTaskSensor`'s `execution_date_fn` parameter to dynamically calculate the dependency's execution date

I've now realized this wouldn't be as great of a solution since `ExternalTaskMarker`s are a factor here as well (though IMO it seems like Airflow shouldn't need them), and they don't appear to have such a way to dynamically calculate the execution date/delta.